src/tokenizer/main.py

```diff
@@ -150,6 +157,9 @@ def val(t, quote_word=False):
     else:
         to_text = lambda t: t.txt
 
+    if args.ospl:
```

Comment:
This would then be `args.onesentperline`

Reply:
Done!